data-prepper-api/src/test/java/com/amazon/dataprepper/model/event/DefaultEventMetadataTest.java

```diff
@@ -93,7 +93,7 @@ public void testBuild_withoutTimeReceived() {
         final Instant timeReceived = result.getTimeReceived();
         assertThat(timeReceived, notNullValue());
         assertThat(timeReceived, is(greaterThan(before)));
-        assertThat(timeReceived, is(lessThan(Instant.now())));
+        assertThat(timeReceived, is(lessThan(Instant.now().plusSeconds(1))));
```

Comment:
I'm guessing you are adding second incase the times are the same. You be able to use `lessThanOrEqualTo` instead.